logpyle/__init__.py

```diff
@@ -69,29 +69,12 @@
 from typing import (List, Callable, Union, Tuple, Optional, Dict, Any,
                     TYPE_CHECKING, Iterable)
 from pytools.datatable import DataTable
+from time import monotonic as time
```

Comment:
```suggestion
from time import monotonic as time_monotonic
```
?

Reply:
Done in fd751accdb795520cead252cb2aa29593fd5dd4c